client/blocks/upsell-nudge/index.jsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import classnames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Banner from 'components/banner';
+import { recordTracksEvent, withAnalytics } from 'state/analytics/actions';
+import { navigate } from 'state/ui/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export const UpsellNudge = ( {
```

Comment:
Do we need to handle default values somehow?

Reply:
Yes, that's probably a good idea! Added defaults for the icon and compact styles; I don't know if it makes sense to add default values for the other props. The Tracks stuff is still up in the air, too, so I'll wait to add defaults there until we've figured out what to do.